src/components/TeamForm/TeamForm.js

```diff
@@ -68,6 +97,30 @@ class TeamForm extends Component {
             onChange={this.handleChange('address')}
           />
         </FormGroup>
+        <FormGroup controlId="teamForm-vote-duration">
+          <ControlLabel>Sort duration</ControlLabel>
+          <OverlayTrigger trigger="focus" placement="right" overlay={popoverRight}>
+            <Button
+              bsSize="xsmall"
+              className="glyphicon glyphicon-question-sign"
+              style={{ margin: 5 }}
+            />
+          </OverlayTrigger>
+          <Row>
+            <Col sm={6}>
```

Comment:
Now that the help text is in a popover, I think this column would look better at `2` instead of `6`.